documentation/generateDocs.js

```diff
@@ -0,0 +1,275 @@
+var fs= require('fs');
+var path = require('path');
+var data=fs.readFileSync( path.join(__dirname, './docmaker.json'), 'utf8');
+var dockmaker_elements=JSON.parse(data);
+
+const { exec } = require('child_process');
+exec('jsdoc -X ./src/[c,e,u]*/*.js > ./documentation/AST/core_AST.json && jsdoc -X ./src/animation.js > ./documentation/AST/animation_AST.json && jsdoc -X ./src/extensions/layout/index.js > ./documentation/AST/layout_AST.json', { "shell": "/bin/bash" },(err, stdout, stderr) => {
```

Comment:
This exec should be a separate npm script (or several scripts) in package.json.  The npm scripts should run before `generateDocs.js`.

That way, the docs can be built on Windows as well as Linux.

Reply:
Thank you for pointing it out. I have fixed it.